detection_rules/main.py

```diff
@@ -278,7 +281,8 @@ def _export_rules(rules: RuleCollection, outfile: Path, downgrade_version: Optio
 @click.option('--skip-unsupported', '-s', is_flag=True,
               help='If `--stack-version` is passed, skip rule types which are unsupported '
                    '(an error will be raised otherwise)')
-def export_rules(rules, outfile: Path, replace_id, stack_version, skip_unsupported) -> RuleCollection:
+@click.option('--include-metadata', type=bool, is_flag=True, default=False, help='Add metadata to the exported rules')
+def export_rules(rules, outfile: Path, replace_id, stack_version, skip_unsupported, include_metadata) -> RuleCollection:
```

Comment:
```suggestion
def export_rules(rules, outfile: Path, replace_id, stack_version, skip_unsupported, include_metadata: bool) -> RuleCollection:
```

Nit, should we add a type hint here?